src/libs/Navigation/types.ts

```diff
@@ -216,8 +216,8 @@ type MoneyRequestNavigatorParamList = {
         threadReportID: string;
     };
     [SCREENS.MONEY_REQUEST.STEP_DESCRIPTION]: {
```

Comment:
I meant to fix the whole types here. e.g. `field` is not param. and add missing props e.g. `backTo`